projects/components/src/metric-card/metric-card.component.ts

```diff
@@ -52,4 +67,35 @@ export class MetricCardComponent {
 
   @Input()
   public borderColor: Color = Color.Transparent;
+
+  @Input()
+  public layoutType: MetricCardLayout = MetricCardLayout.CardWithIndicator;
+
+  @Input()
+  public iconColor: Color = Color.Gray9;
+
+  @Input()
+  public isSelected: boolean = false;
+
+  @Input()
+  public clickable: boolean = false;
```

Comment:
we probably don't need clickable property. Parent component can decide to put `(click)` handler directly over this component host

Reply:
yeah, removed